cg/cli/workflow/rnafusion/base.py

```diff
@@ -104,7 +105,7 @@ def run(
     analysis_api: RnafusionAnalysisAPI = context.meta_apis[MetaApis.ANALYSIS_API]
     analysis_api.status_db.verify_case_exists(case_internal_id=case_id)
 
-    command_args: CommandArgs = CommandArgs(
+    command_args: NfCommandArgs = NfCommandArgs(
```

Comment:
Similarly here, although I see that this is a different pipeline. 
@fevac , @sofstam can you also have a look at the comment above regarding the building of the `NfCommandArgs` within the `AnalysisAPI` instead of the CLI layer for RNAfusion and Taxprofiler